vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/resources/META-INF/resources/frontend/lumo-utility.ts

```diff
@@ -0,0 +1,15 @@
+import { utility } from '@vaadin/vaadin-lumo-styles/utility.js';
+
+var hasStyles = false;
+const styleTags = document.head.getElementsByTagName('style');
+for (var i = 0; i < styleTags.length; i++) {
+  if (styleTags[i].sheet === utility.styleSheet) {
+    hasStyles = true;
+    break;
+  }
+}
+if (!hasStyles) {
```

Comment:
I'm still getting duplicate style declarations when enabling the utility module in `theme.json`. I'm not sure if the check here works correctly with the `injectGlobalCss` generated by Flow, which creates new stylesheets rather than reusing imported one. The other problem is that `injectGlobalCss` uses a different duplicate check than the one implemented here, so if `injectGlobalCss` runs after this code, you will get duplicates anyway.

Reply:
Should this be handled by documentation? I.e. by telling not to use importing via theme.json when using Flow.

Reply:
With the latest changes in Flow, you can remove this TS file and instead import `utility-global` in the Java file. As Flow imports this same file, you should not get duplicates anymore

Reply:
Done